ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -475,6 +490,19 @@ func (s *Service) GetTags() ([]*ecs.Tag, error) {
 	return s.tags, nil
 }
 
+func (s *Service) getLoadBalancerList(loadBalancerValue []string) ([]*ecs.LoadBalancer, error) {
+	if s.loadBalancers == nil {
+		loadBalancers := make([]*ecs.LoadBalancer, 0)
```

Comment:
Usually in golang we prefer to init an empty array like this

```
var loadBalancers []*ecs.LoadBalancer
```

unless you know the length of the slice you can do

```
loadBalancers := make([]*ecs.LoadBalancer, len(loadBalancerValue)
```

Refer to https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices